src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from "react";
+
+function Login() {
+  const [username, setUsername] = useState("");
```

Comment:
**suggestion**: Add type annotations for event handlers to enhance type safety and readability.